webpush_channels/tests/test_views_channel.py

```diff
@@ -114,7 +115,7 @@ def setUp(self):
         self.webpusher_error_patcher = mock.patch('webpush_channels.views.channels.WebPusher')
 
     def test_push_notifications_can_be_sent_to_channel_with_registration_and_subscription(self):
-        data = "You have a message"
+        data = None
```

Comment:
Be aware that `json.dumps(None)` will return 'null'. Is it the message we want to send through as the WebPush payload?